### PR TITLE
quickstart/{java, go}/metrics: Add tags for "status" and "error"

### PR DESCRIPTION
**Pull request by hvent90**
Addresses #470

Part 1 of 2 pull request to reduce cognitive load as reviewing quickstarts is no simple task.

This PR does the following for Go & Java Metric quickstarts:

1. Removes the "error" measure
2. Creates a tag for "status" and "Error"
3. Sets "status" tag to "OK"
4. If error, "status" tag is set to "ERROR" and "error" tag is set to the error string
5. Wrapped `readEvaluateProcessLine` in a latency measurement to catch errors

### CODE REVIEW

File: content/quickstart/go/metrics.md [hunk 1]
```diff
@@ -406,15 +401,17 @@ Now we will insert a specific tag called "repl". It will give us a new `context.
 
 For example
 ```go
-        ctx, err := tag.New(context.Background(), tag.Insert(KeyMethod, "repl"))
+ctx, _ := tag.New(context.Background(), tag.Insert(KeyMethod, "repl"),
```
Comment (odeke-em) [inline content/quickstart/go/metrics.md]: Please run these Go snippets through go-fmt first.

File: content/quickstart/go/metrics.md [hunk 2]
```diff
@@ -406,15 +401,17 @@ Now we will insert a specific tag called "repl". It will give us a new `context.
...
 ```
 
 and for complete usage:
 
 {{<tabs Snippet All>}}
 {{<highlight go>}}
-func readEvaluateProcess(br *bufio.Reader) error {
-	ctx, err := tag.New(context.Background(), tag.Insert(KeyMethod, "repl"))
+func readEvaluateProcess(br *bufio.Reader) (terr error) {
+  ctx, _ := tag.New(context.Background(), tag.Insert(KeyMethod, "repl"),
```
Comment (odeke-em) [inline content/quickstart/go/metrics.md]: Ditto about go-fmt

File: content/quickstart/go/metrics.md (hunk 1 above)
Comment (hvent90) [reply content/quickstart/go/metrics.md]: thank you!

**Comment by hvent90** [comment]
@odeke-em indentation fixed in latest commit. I also added a bit of noise to the PR by switching from shortcodes to backticks ` ``` ` for my editor (Atom) to correctly indent the files.

**Pull request merged**